src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixActionsSource.cs

```diff
@@ -141,11 +143,21 @@ public void Dispose()
             issueLocationsTagAggregator.Dispose();
         }
 
-        private async void TagAggregator_TagsChanged(object sender, TagsChangedEventArgs e)
```

Comment:
FYI VSTHRD100 warns about `async void` methods - we have eight other occurrences.
Fixing those is covered by #3171, although we might decide to break that down into separate tickets when we start working on it.